packages/react/src/patterns/sub-patterns/TableOfContents/TableOfContents.js

```diff
@@ -152,8 +170,8 @@ const TableOfContents = ({ menuItems, children, menuLabel, theme }) => {
           className={`${prefix}--tableofcontents__sidebar`}
           data-sticky="true">
           <div className={`${prefix}--tableofcontents__mobile-top`}></div>
-          <TOCDesktop {...props} />
-          <TOCMobile {...props} />
+          <TOCDesktop autoMenuItems={autoMenuItems} {...props} />
+          <TOCMobile autoMenuItems={autoMenuItems} {...props} />
```

Comment:
What is `autoMenuItems`? Is this even necessary? They don't appear to be props for TOCDecktop or TOCMobile unless I missed something?